app/pages/landingIndustryUser.js

```diff
@@ -0,0 +1,220 @@
+import React, { Component } from 'react';
+import { Button, Accordion, Card, Container, Col, Row } from 'react-bootstrap';
+import Header from '../components/Header';
+import Footer from '../components/Footer';
+import { graphql } from 'react-relay';
+import Organisations from '../containers/Organisations/Organisations';
+
+const organizations = [
+  { orgName: 'Rio Tinto Alcan', facilities: ['Facility1', 'Facility2'] },
+  {
+    orgName: 'Shell Coporation',
+    facilities: ['Facility1', 'Facility2', 'Facility3']
+  },
+  { orgName: 'Shanghai Eletric', facilities: ['Facility1', 'Facility2'] }
+];
+
+class landingIndustryUser extends Component {
```

Comment:
```suggestion
class LandingIndustryUser extends Component {
```

Reply:
This component should be moved to the pageContainers folder, and the current file should only import the component defined in the pageContainers folder. This is to work around the fact that we cannot have graphql queries in the `pages` directory, as this will generate modules that are not pages, and Next isn't happy about that